cmd/ascii2der/scanner.go

```diff
@@ -284,6 +284,53 @@ again:
 		if s.pos.Offset+1 < len(s.text) && s.text[s.pos.Offset+1] == '"' {
 			return s.parseUTF32String()
 		}
+	case 'b':
+		if s.pos.Offset+1 < len(s.text) && s.text[s.pos.Offset+1] == '`' {
+			s.advance() // Skip the b.
+			s.advance() // Skip the `.
+			bitStr, ok := s.consumeUpTo('`')
+			if !ok {
+				return token{}, &parseError{s.pos, errors.New("unmatched `")}
+			}
+
+			// The leading byte is the number of "extra" bits at the end.
+			var bitCount int
+			var sawPipe bool
+			bits := []byte{0}
```

Comment:
Nitpick: `bits` made me read this as a `[]bool` when it's not. :-) `bytes` is a module name, so how about `value`?

Reply:
It's a `vector<bool>`. =P